be/src/vec/exprs/table_function/vexplode_json_array.cpp

```diff
@@ -144,7 +143,8 @@ void VExplodeJsonArrayTableFunction<DataImpl>::_insert_values_into_column(Mutabl
         size_t old_size = nullmap_column->size();
         nullmap_column->resize(old_size + max_step);
         memcpy(nullmap_column->get_data().data() + old_size,
-               _parsed_data.get_null_flag_address(_cur_offset), max_step * sizeof(UInt8));
+               _parsed_data.get_null_flag_address(cast_set<int>(_cur_offset)),
```

Comment:
why here need cast？ not use cast_set  to do the work where here really need int?